tests/test_elasticsearch.py

```diff
@@ -79,22 +79,46 @@ def test_elasticsearch_with_sysctl_disabled(self, kube_version):
         )
 
         assert len(docs) == 3
-        doc = docs[0]
 
         # elasticsearch master
+        doc = docs[0]
         assert doc["kind"] == "StatefulSet"
-        assert "sysctl" not in jmespath.search(
-            "spec.template.spec.initContainers[*].name", docs[0]
-        )
+        assert jmespath.search("spec.template.spec.initContainers", docs[0]) is None
 
         # elasticsearch data
         doc = docs[1]
         assert doc["kind"] == "StatefulSet"
-        assert "sysctl" not in jmespath.search(
-            "spec.template.spec.initContainers[*].name", docs[1]
-        )
+        assert jmespath.search("spec.template.spec.initContainers", docs[1]) is None
 
         # elasticsearch client
         doc = docs[2]
         assert doc["kind"] == "Deployment"
         assert jmespath.search("spec.template.spec.initContainers", docs[2]) is None
+
+    def test_elasticsearch_securitycontext_defaults(self, kube_version):
```

Comment:
Because this test case is all defaults, it's basically the same as the first test case in this file, so you could take these assertions and put them into that "default" test case, and just comment on what is being tested, or do it without comments since the meaning is implied by the code.